assets/js/components/includes/DropdownFilter.vue

```diff
@@ -44,23 +46,17 @@ export default Vue.extend({
   },
 
   computed : {
-
+    current() {
+      return FILTERS[this.filter];
     },
+  },
 
   methods: {
-    getIndex(){
-      let select = event.target;
```

Comment:
event is undefined. if it works in your browser, it's just a quirk. it *is* undefined.